app/src/features/apiClient/screens/apiClient/components/sidebar/components/sidebarListHeader/sidebarListHeader.scss

```diff
@@ -1,15 +1,15 @@
 .sidebar-list-header {
   display: flex;
-  gap: 6px;
   align-items: center;
+  gap: 6px;
   padding: 8px 0;
 
   &-button {
     padding: 4px 8px;
   }
 
   &-search {
-    width: 200px;
+    width: -webkit-fill-available;
```

Comment:
Why no `100%`?

Reply:
The list header can also have a 3 dot dropdown menu on the right hand side of the input. I've changed this to `flex:1` though.